Gems/Atom/Asset/Shader/Code/AZSL/Platform/Linux/Vulkan/AzslcHeader.azsli

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+/*
+* The shader build pipeline has 2 preprocess stages. The first one happens 
+* after appending this header and pre-processing with mcpp ahead of azslc.
+*/
+
+//Used for normal map sampling
+#define AZ_TRAIT_ASTC_COMPRESSION 1
```

Comment:
I don't have that in my branch, I copied the file from Windows, this looks like it comes from Android which needs that define?

Reply:
yes. We dont need to enable AZ_TRAIT_ASTC_COMPRESSION for Linux as you should be able to use BCn compressions similar to PC

Reply:
Will remove this define